lib/version_utils.pm

```diff
@@ -576,7 +576,7 @@ sub get_os_release {
     my ($go_to_target, $os_release_file) = @_;
     $go_to_target    //= '';
     $os_release_file //= '/etc/os-release';
-    my %os_release = script_output("$go_to_target cat $os_release_file") =~ /^(\S+)="?([^"\r\n]+)"?$/gm;
+    my %os_release = script_output("$go_to_target cat $os_release_file") =~ /^([^#]\S+)="?([^"\r\n]+)"?$/gm;
```

Comment:
```suggestion
    my %os_release = script_output("$go_to_target cat $os_release_file") =~ /^#?(\S+)="?([^"\r\n]+)"?$/gm;
```

Reply:
i think this is wrong. the purpose is to not match the lines which start with `#`. `^#?(\S+)` will match anything that starts or not with hash. i will resolve this and feel free to open it if you think i am wrong.